src/sentry/grouping/enhancer/enhancement-configs/mobile@2021-04-02.txt

```diff
@@ -125,6 +125,8 @@ module:android.view.* category=ui
 module:android.text.* category=ui
 module:android.widget.* category=ui
 module:android.app.Dialog category=ui
+module:androidx.*.widget.* category=ui
```

Comment:
```suggestion
module:androidx.*.widget.* category=ui
module:androidx.*.view.* category=ui
```

since we have `android.view.*`, makes sense to add `androidx.*.view.*` too

Reply:
@marandaneto do you have an example event for that?